src/python/pants/option/subsystem.py

```diff
@@ -96,21 +97,32 @@ def get_scope_info(cls) -> ScopeInfo:
         return cls.create_scope_info(scope=cls.options_scope, subsystem_cls=cls)
 
     @classmethod
+    @deprecated(
+        removal_version="2.12.0.dev0",
+        hint=(
+            "Options are now registered by declaring class attributes using the types in "
+            "pants/option.option_types.py"
```

Comment:
Probably link to `doc_url('plugin-upgrade-guide')`

Reply:
Also: "rather than by using the class-method `register_options`."

Reply:
pants/option.option_types.py is a typo, should be pants/option/option_types.py

but really this should link to a doc url, this help message is not enough info.

Reply:
@thejcannon, you can fold this in to some other upcoming related change, to save a tree

Reply:
I have no idea how I missed this thread :see_no_evil:

Reply:
I got here going to apologize for the late comment on the typo, but see it was spotted and overlooked at the same time :P  (GH is horrible when it comes to PR threads, I'm amazed we don't miss more..)

Reply:
https://github.com/pantsbuild/pants/pull/14786

Reply:
For having the same parent company I'm surprised there isn't more cross-pollination between Azure DevOps and GitHub. AzDO has lots of really great features like blocking merge until comments are resolved and expiring checks after a certain amount of time.